### PR TITLE
using thing

### PR DESCRIPTION
**Pull request by SomeHybrid**
fixing #9 (also i didnt test it bc circular imports shit and docker brokey)

### CODE REVIEW

File: src/bot/exts/fun/uwu.py [hunk 1]
```diff
@@ -199,6 +60,6 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
         await ctx.send(content=converted_text, embed=embed)
 
 
-async def setup(bot: Bot) -> None:
+async def setup(bot: commands.Bot) -> None:
```
Comment (shenanigansd) [inline src/bot/exts/fun/uwu.py:63]:
```suggestion
async def setup(bot: Bot) -> None:
```
This `Bot` is the subclass defined in `bot/bot.py`, not the base Discord.py `commands.bot`.

**Review by shenanigansd — CHANGES_REQUESTED** [review]
Everything looks good at first glance.
You'll also need to add the new dependency to the `pyproject.toml`

File: src/bot/exts/fun/uwu.py [hunk 2]
```diff
@@ -199,6 +60,6 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
         await ctx.send(content=converted_text, embed=embed)
```
Comment (Robin5605) [inline src/bot/exts/fun/uwu.py:60]: See above
```suggestion
        await ctx.send(content=converted_text, embeds=embeds)
```

File: src/bot/exts/fun/uwu.py [hunk 3]
```diff
@@ -179,18 +40,18 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
 
         # Grabs the text from the embed for uwuification
         if embeds:
-            embed = messages.convert_embed(self._uwuify, embeds[0])
+            embed = messages.convert_embed(imsosorry.uwuify, embeds[0])
```
Comment (Robin5605) [inline src/bot/exts/fun/uwu.py:43]: Handle multiple embeds
```suggestion
            embeds = [messages.convert_embed(imsosorry.uwuify, embed) for embed in embeds]
```

File: src/bot/exts/fun/uwu.py [hunk 4]
```diff
@@ -179,18 +40,18 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
 
         # Grabs the text from the embed for uwuification
         if embeds:
```
Comment (Robin5605) [inline src/bot/exts/fun/uwu.py:42]: No longer needed

**Review by Robin5605 — APPROVED** [review]

**Pull request closed without merging**